getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
+nodes and authenticate to them.
```

Comment:
```suggestion
nodes for appropriate authentication.
```
Not sure what you were going for, so I took a guess?

Reply:
FWIW, I think @solugebefola's suggestion is good.

Reply:
Elaborated a bit to make it more clear.